front/src/modules/rollingStock/components/RollingStockCard/RollingStockCardDetail.tsx

```diff
@@ -72,121 +72,119 @@ export default function RollingStockCardDetail({
+      <tbody>
+        <tr>
+          <td className="text-primary">{t('startupTime')}</td>
+          <td>
+            {rollingStock.startup_time}
+            <span className="small ml-1 text-muted">s</span>
+          </td>
+        </tr>
+        <tr>
+          <td className="text-primary">{t('startupAcceleration')}</td>
+          <td>
+            {rollingStock.startup_acceleration}
+            <span className="small ml-1 text-muted">m/s²</span>
+          </td>
+        </tr>
+        <tr>
+          <td className="text-primary">{t('comfortAcceleration')}</td>
+          <td>
+            {rollingStock.comfort_acceleration}
+            <span className="small ml-1 text-muted">m/s²</span>
+          </td>
+        </tr>
+        <tr>
+          <td className="text-primary">{t('inertiaCoefficient')}</td>
+          <td>{rollingStock.inertia_coefficient}</td>
+        </tr>
+        <tr>
+          <td className="text-primary">{t('gammaValue')}</td>
+          <td>
+            {rollingStock.gamma.value * -1}
+            <span className="small ml-1 text-muted">m/s²</span>
+          </td>
+        </tr>
+        <tr>
+          <td className="text-primary">{t('loadingGauge')}</td>
+          <td>{rollingStock.loading_gauge}</td>
+        </tr>
+        <tr>
+          <td className="text-primary">{t('basePowerClass')}</td>
+          <td>{rollingStock.base_power_class}</td>
+        </tr>
+      </tbody>
+    </table>
+  );
+  const rightColumn = rollingStock && (
+    <table className="rollingstock-details-table">
+      <tbody>
+        <tr>
+          <td className={cx({ formResistance: form })}>{t('rollingResistance')}</td>
+          <td className={cx('text-primary text-muted', { formResistance: form })}>
+            {t('rollingResistanceFormula')}
+          </td>
+        </tr>
+        <tr>
+          <td className="text-primary">{t('rollingResistanceA')}</td>
+          <td>
+            {Math.floor(
+              rollingStock.rolling_resistance?.A
+                ? (rollingStock.rolling_resistance.A * 10000) / 10000
+                : 0
+            )}
+            <span className="small ml-1 text-muted">N</span>
+          </td>
+        </tr>
+        <tr>
+          <td className="text-primary">{t('rollingResistanceB')}</td>
+          <td>
+            {Math.floor(
+              rollingStock.rolling_resistance?.B
+                ? (rollingStock.rolling_resistance.B * 10000) / 10000
+                : 0
+            )}
+            <span className="small ml-1 text-muted">N/(m/s)</span>
+          </td>
+        </tr>
+        <tr>
+          <td className="text-primary">{t('rollingResistanceC')}</td>
+          <td title={rollingStock.rolling_resistance?.C.toString()}>
+            {floor(rollingStock.rolling_resistance?.C ?? 0, 2)}
+            <span className="small ml-1 text-muted">N/(m/s)²</span>
+          </td>
+        </tr>
+        {rollingStock.supported_signaling_systems &&
```

Comment:
signaling_systems cannot be undefined :) 
```suggestion
        {
```